src/ThreeDSAuth.res

```diff
@@ -5,7 +5,7 @@ let make = () => {
   let (openModal, setOpenModal) = React.useState(_ => false)
   let (loader, setloader) = React.useState(_ => true)
 
-  let logger = Recoil.useRecoilValueFromAtom(RecoilAtoms.loggerAtom)
+  let logger = OrcaLogger.make()
```

Comment:
Why are we creating a new logger instead of using the same from Recoil?

Reply:
This is a separate iframe.
Recoil.useRecoilValueFromAtom(RecoilAtoms.loggerAtom) returns a dummy instance of iframe